config/phpstan/parser.neon

```diff
@@ -1,3 +1,6 @@
+parameters:
+    phpVersion: 80100
```

Comment:
could you check this when used in scoped version running php 7.x? You may can clone rector/rector and check against this directory:

https://github.com/rectorphp/rector/blob/5e0eef55a50c696ddb922556061c09f2a1570b58/.github/workflows/e2e.yaml#L21

Reply:
Seems ok:
```
root@99f6d3b4e61d:/code/rector/e2e/define-constant# ../../bin/rector process --dry-run --ansi

 1/1 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 [OK] Rector is done!


root@99f6d3b4e61d:/code/rector/e2e/define-constant# php -v
PHP 7.4.25 (cli) (built: Oct 22 2021 17:23:33) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies

root@99f6d3b4e61d:/code/rector/e2e/define-constant# git diff
diff --git a/config/phpstan/parser.neon b/config/phpstan/parser.neon
index 58a2839..2504a37 100644
--- a/config/phpstan/parser.neon
+++ b/config/phpstan/parser.neon
@@ -1,3 +1,6 @@
+parameters:
+    phpVersion: 80100
+
 services:
     defaultAnalysisParser:
         factory: @cachedRectorParser
root@99f6d3b4e61d:/code/rector/e2e/define-constant#
```

Reply:
Thank you for verify